rs/nns/governance/src/pb/conversions.rs

```diff
@@ -2900,7 +2900,12 @@ impl From<pb::InstallCode> for pb_api::InstallCode {
         let wasm_module_hash = item
             .wasm_module
             .map(|wasm_module| super::calculate_hash(&wasm_module).to_vec());
-        let arg_hash = item.arg.map(|arg| super::calculate_hash(&arg).to_vec());
+        let arg = item.arg.unwrap_or_default();
+        let arg_hash = if arg.is_empty() {
+            Some(vec![])
```

Comment:
It feels weird when there's no arg to use Some(vec![]) for the arg hash, instead of None (since there's nothing to check)?

Reply:
As mentioned in the PR description, since the `ic0.install_code` eventually takes a `Vec<u8>` (instead of `Option<Vec<u8>>`). I think if we use `None` to represent it, the weirdness will still exist somewhere (e.g. when executing, we do `unwrap_or_default()`. It seems simpler to me if we have one representation throughout the entire flow.

Reply:
I see the symmetry there between arg and arg_hash.  I don't have super strong feelings about it either way